src/components/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Paper, Container } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+const styles = (theme) => ({
+  content: {
+    flexGrow: 1,
+  },
+  description: {
+    padding: theme.spacing(2),
+    textAlign: 'left',
+    color: theme.palette.text.secondary,
+  },
+  image: {
+    maxHeight: '800px',
+    width: '70%',
+  },
+  blogSide: {
+    marginTop: '120px',
+  },
+});
+
+function BlogPost(props) {
+  const { classes } = props;
+
+  return (
+    <Container>
+      <Grid container spacing={3}>
+        <Grid className={classes.blogSide} item xs={3}>
```

Comment:
Based on your website, it looks like on devices smaller than medium, it stacks like a column
![Screen Shot 2020-06-08 at 8 26 51 PM](https://user-images.githubusercontent.com/38300070/84096027-703ebd00-a9c6-11ea-9194-013c73291a28.png). You can accomplish this by giving two different col size to the grid item. For example ```<Grid item xs={12} md={3}>```. From there, you will see that this blogside will be on top of the post so you are going to have to change the order of these items based on screen size. Use this for reference
https://stackoverflow.com/questions/54726871/changing-the-order-of-grid-item-stacking-in-material-ui